src/containers/Review/assignment/AssignAll.tsx

```diff
@@ -56,11 +65,12 @@ const getReviewerList = (assignments: AssignmentDetails[]) => {
       const {
         reviewer: { id, firstName, lastName },
       } = assignment
+      const displayName = id === currentUserId ? selfString : `${firstName} ${lastName}`
```

Comment:
Added so it displays "Yourself" for the current user, for consistency with the section selectors.